src/components/Todo.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import TodoForm from "./TodoForm";
+import { RiCloseCircleLine } from "react-icons/ri";
+import { TiEdit } from "react-icons/ti";
+
+function Todo({ todos, completeTodo, removeTodo, updateTodo }) {
+  const [editMode, setEditMode] = useState(false);
+  const [edit, setEdit] = useState({
+    id: null,
+    value: "",
+  });
+
+  const submitUpdate = (value) => {
+    updateTodo(value.id, value);
+
+    setEdit({
+      id: null,
+      value: "",
+    });
+  };
+  if (edit.id) {
+    return <TodoForm edit={edit} editMode={editMode} setEditMode={setEditMode} onSubmit={submitUpdate} />;
+  }
+  return todos.map((todo, index) => (
+    <div
+      className={todo.isCpmplete ? "todo-row complete" : "todo-row"}
+      key={index}
+    >
+      <div key={todo.id} onClick={() => completeTodo(todo.id)}>
+        {todo.text}
+      </div>
+      <div className="icons">
+        <RiCloseCircleLine
+          onClick={() => removeTodo(todo.id)}
+          className="delete - icon"
```

Comment:
with the spacing in the className, this styling will probably not be applied properly